### PR TITLE
move to separate prop-types package. upgrade webpack and babel. remov…

### PR DESCRIPTION
**Pull request by kentmclean**
…e unnused dependencies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,52 +1,43 @@
...
-    "url": "https://github.com/CanopyTax/cpr-multiselect/issues"
-  },
-  "homepage": "https://github.com/CanopyTax/cpr-multiselect",
-  "devDependencies": {
-    "autoprefixer-loader": "1.2.0",
-    "babel": "5.8.23",
-    "babel-core": "5.8.25",
-    "babel-loader": "5.3.2",
-    "css-loader": "^0.20.2",
-    "jasmine-core": "2.3.4",
```
Comment (kentmclean) [inline package.json:30]: the testing dependencies were probably copy/pasted from another repo. There aren't any tests defined within this repo, so I removed the test-related dev dependencies

File: package.json [hunk 2]
```diff
@@ -1,52 +1,43 @@
...
+		"babel-preset-es2015": "^6.24.1",
+		"babel-preset-react": "^6.24.1",
+		"babel-preset-stage-1": "^6.24.1",
+		"css-loader": "^0.28.5",
+		"postcss-loader": "^2.0.6",
+		"prop-types": "^15.5.10",
+		"react": "^15.6.1",
+		"react-dom": "^15.6.1",
+		"style-loader": "^0.13.0",
+		"webpack": "^3.5.5"
```
Comment (kentmclean) [inline package.json:37]: upgraded webpack

File: package.json [hunk 3]
```diff
@@ -1,52 +1,43 @@
...
+		"url": "https://github.com/CanopyTax/cpr-multiselect.git"
+	},
+	"author": "Bret Little",
+	"license": "MIT",
+	"bugs": {
+		"url": "https://github.com/CanopyTax/cpr-multiselect/issues"
+	},
+	"homepage": "https://github.com/CanopyTax/cpr-multiselect",
+	"devDependencies": {
+		"babel": "^6.23.0",
```
Comment (kentmclean) [inline package.json:24]: upgraded babel

File: src/multi-selector.js [hunk 4]
```diff
@@ -11,18 +12,20 @@ function DefaultItemComponent(props) {
...
 		</div>
-	)
+	);
 }
 
 DefaultItemComponent.propTypes = {
-	item: React.PropTypes.any.isRequired,
-	getItemTitle: React.PropTypes.func,
-	selectedItems: React.PropTypes.array.isRequired
+	item: PropTypes.any.isRequired,
```
Comment (kentmclean) [inline src/multi-selector.js:26]: these are literally the only substantive changes within this file. All others are prettier doing its thing.

**Review by oklandon — APPROVED** [review]

**Pull request merged**